openshift/e2e-common.sh

```diff
@@ -144,75 +170,25 @@ function install_knative_eventing(){
 
 function run_e2e_tests(){
   header "Running tests with Multi Tenant Channel Based Broker"
+
+  local test_name=$1
+  local failed=0
+  local channels=messaging.knative.dev/v1alpha1:InMemoryChannel,messaging.knative.dev/v1alpha1:Channel,messaging.knative.dev/v1beta1:InMemoryChannel
```

Comment:
this is wrong, see: 
https://github.com/openshift/knative-eventing/pull/646/files#diff-cd758740b9c6da1c23340da7084242feL150

```
-channels=messaging.knative.dev/v1beta1:Channel,messaging.knative.dev/v1beta1:InMemoryChannel
```

The `v1alpha1` has been removed

Reply:
Updated. Thanks.